pkg/controllers/vdb/imageversion_reconciler.go

```diff
@@ -108,6 +109,17 @@ func (v *ImageVersionReconciler) Reconcile(ctx context.Context, _ *ctrl.Request)
 	return v.verifyNMADeployment(vinf, pod)
 }
 
+// genSandboxOldVersionInfo will build and return the sandbox in the configmap
+// annotations
+func (v *ImageVersionReconciler) genSandboxOldVersionInfo(ctx context.Context) (*version.Info, error) {
+	sbTrigger := MakeSandboxTrigger(v.Rec, v.Vdb, v.PFacts.SandboxName, "" /*no uuid*/)
+	oldVersion, err := sbTrigger.getSandboxVersion(ctx)
```

Comment:
Does this handle the case where the version annotation isn't present? I think to be consistent that should be tolerated.

Reply:
Got you!